client/my-sites/marketplace/pages/marketplace-domain-upsell/index.tsx

```diff
@@ -97,7 +97,7 @@ function CalypsoWrappedMarketplaceDomainUpsell(): JSX.Element {
 
 	const redirectToUseDomainFlow = (): void => {
 		const currentUrl = '/plugins/domain';
-		const useYourDomainUrl = `/start/new-launch/domains-launch/use-your-domain?siteSlug=${ selectedSite?.slug }&source=${ currentUrl }`;
+		const useYourDomainUrl = `/start/launch-site/domains-launch/use-your-domain?siteSlug=${ selectedSite?.slug }&source=${ currentUrl }`;
```

Comment:
Note to self: Confirm if redirecting to `/start/launch-site/...` is correct and whether `source` is still needed in the query string (not defined in `launch-site` deps).